Cargo.lock

```diff
@@ -1696,10 +1696,10 @@ dependencies = [
 
 [[package]]
 name = "unicase"
-version = "2.5.1"
+version = "2.6.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
- "version_check 0.1.5 (registry+https://github.com/rust-lang/crates.io-index)",
+ "version_check 0.9.1 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
I wonder if they accept a PR replacing `version_check` with `auto-cfg`? It seems like the latter is positions to be the winner crate in this domain